analyzer/analyzer.go

```diff
@@ -16,23 +16,41 @@ import (
 	"golang.org/x/tools/go/analysis"
 )
 
+type optionInt struct {
+	enabled bool
+	intConv bool
+}
+
+type optionErr struct {
+	enabled  bool
+	errError bool
+	errorf   bool
+}
+
+type optionStr struct {
+	enabled   bool
+	sprintf1  bool
+	strconcat bool
+}
```

Comment:
I would prefer `activated` instead of `enabled`, because `disabled` is connoted for people with disabilities


```suggestion
type optionInt struct {
	activated bool
	intConv   bool
}

type optionErr struct {
	activated  bool
	errError   bool
	errorf     bool
}

type optionStr struct {
	activated  bool
	sprintf1   bool
	strconcat  bool
}
```